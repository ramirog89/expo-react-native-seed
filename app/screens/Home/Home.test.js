import React from "react";
import renderer from "react-test-renderer";

import Home from "./Home";

describe("<Home />", () => {
  it("should render", () => {
    const rendered = renderer.create(<Home />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
