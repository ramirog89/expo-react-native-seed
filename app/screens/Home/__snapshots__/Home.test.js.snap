// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginBottom": 20,
        "marginTop": 10,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 80,
          "marginLeft": -10,
          "marginTop": 3,
          "resizeMode": "contain",
          "width": 100,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginHorizontal": 50,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "rgba(96,100,109, 1)",
          "fontSize": 17,
          "lineHeight": 24,
          "textAlign": "center",
        }
      }
    >
      React Native Starter
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginTop": 15,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
          "paddingVertical": 15,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#2e78b7",
            "fontSize": 18,
            "fontWeight": "bold",
          }
        }
      >
        Github repo!
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#eee",
        "bottom": 0,
        "height": 50,
        "justifyContent": "center",
        "position": "absolute",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "rgba(96,100,109, 1)",
          "fontSize": 17,
          "textAlign": "center",
        }
      }
    >
      @nextonlabs
    </Text>
  </View>
</View>
`;
